DEVELOPMENT.md

```diff
@@ -2,48 +2,28 @@
 
 - [Developer's Guide](#developers-guide)
   - [Setting Up Your macOS Development Environment](#setting-up-your-macos-development-environment)
-  - [Running Scripts with `hatch`](#running-scripts-with-hatch)
+  - [Testing and Linting](#testing-and-linting)
   - [Installing Pre-Commit Hooks](#installing-pre-commit-hooks)
-  - [Building the `phoenix` Package](#building-the-phoenix-package)
-  - [Installing a `phoenix` Build](#installing-a-phoenix-build)
+  - [Building the Package](#building-the-package)
+  - [Installing a Phoenix Build](#installing-a-phoenix-build)
+  - [Installing a `git` Branch on Colab](#installing-a-git-branch-on-colab)
   - [Setting Up Your Windows Test Environment](#setting-up-your-windows-test-environment)
     - [Selecting a Virtualization Option](#selecting-a-virtualization-option)
-    - [Installing Python and `phoenix`](#installing-python-and-phoenix)
+    - [Installing Python and Phoenix](#installing-python-and-phoenix)
     - [Configuring a Remote Interpreter](#configuring-a-remote-interpreter)
     - [Troubleshooting](#troubleshooting)
   - [Publishing a New Release](#publishing-a-new-release)
 
 ## Setting Up Your macOS Development Environment
 
-This section shows you how to set up an isolated virtual environment using `pyenv` and `virtualenvwrapper`. If you are new to `pyenv`, you can install it via `brew` with
+We recommend using a virtual environment to isolate your Python dependencies. This guide will use `conda` installed via Miniconda, but you can use a different virtual environment management tool if you want.
 
-```bash
-brew install pyenv
-```
-
-Next, install a `phoenix`-supported Python version, e.g., `3.10.8`, with
-
-```bash
-export PHOENIX_PYTHON_VERSION=<your-supported-python-version>
-pyenv install $PHOENIX_PYTHON_VERSION
-```
-
-Set the global `pyenv` version with
-
-```bash
-pyenv global $PHOENIX_PYTHON_VERSION
-```
+First, ensure that your virtual environment manager is installed. You can install `conda` via Miniconda by following these [instructions](https://docs.conda.io/projects/miniconda/en/latest/#quick-command-line-install) for the appropriate operating system.
 
-Install `virtualenvwrapper` with
+Create a new virtual environment with a Phoenix-compatible Python version. For example,
 
 ```bash
-pip install virtualenvwrapper
-```
-
-Create a new virtual environment with
-
-```bash
-mkvirtualenv phoenix-env
+conda create --name phoenix python=3.10
```

Comment:
should it be 3.8 instead because that's our minimum? note that e.g. pythonRepl tool in langchain needs 3.9
```suggestion
conda create --name phoenix python=3.8
```

Reply:
Agreed